starter-apps/vue/vue-template/src/views/About/About1.vue

```diff
@@ -0,0 +1,17 @@
+<script lang="ts" setup>
+import { useI18n } from 'vue-i18n'
+import Heading from '@/components/Heading.vue'
+import Text from '@/components/Text.vue'
```

Comment:
If we export all internal components in an index file, let's also update these imports accordingly.